akka-docs/src/main/paradox/stream/operators/Source/repeat.md

```diff
@@ -1,20 +1,33 @@
 # Source.repeat
 
-Stream a single object repeatedly
+Stream a single object repeatedly.
 
 @ref[Source operators](../index.md#source-operators)
 
-@@@div { .group-scala }
-
 ## Signature
 
-@@signature [Source.scala](/akka-stream/src/main/scala/akka/stream/scaladsl/Source.scala) { #repeat }
-
-@@@
+@apidoc[Source.repeat](Source$) { scala="#repeat[T](element:T):akka.stream.scaladsl.Source[T,akka.NotUsed]" java="#repeat(T)" }
 
 ## Description
 
-Stream a single object repeatedly
+This source emits a single element repeatedly. It never completes, so the stream will in most cases
+be limited by other means.
```

Comment:
Should it rather say "It never completes, if you want the stream to be finite you will need to limit it buy combining with another operator" or something along those lines?

Reply:
if you want the stream to be finite you will need to limit it buy me a beer